src/components/article/layout/title-row-above.js

```diff
@@ -157,15 +155,25 @@ TitleRowAbove.defaultProps = {
   subtitle: '',
   topicName: '',
   topicSlug: '',
-  theme: constPageThemes.defaultTheme
+  styles: {}
 }
 
 TitleRowAbove.propTypes = {
   subtitle: PropTypes.string,
   title: PropTypes.string.isRequired,
   topicName: PropTypes.string,
   topicSlug: PropTypes.string,
-  theme: constPropTypes.theme
+  styles: PropTypes.shape({
```

Comment:
Since `TitleRowAbove.propTypes.styles`, `ArticleContainer.propTypes.styles` and `TitleRowUpon.propTypes.styles` are a little different, we don't abstract them out.